xbmc/interfaces/legacy/ListItem.h

```diff
@@ -213,13 +214,14 @@ namespace XBMCAddon
        *     - duration      : integer (245) - duration in seconds
        *     - studio        : string (Warner Bros.)
        *     - tagline       : string (An awesome movie) - short description of movie
-       *     - writer        : string (Robert D. Siegel)
+       *     - writer        : string (Robert D. Siegel) - writing credits
        *     - tvshowtitle   : string (Heroes)
        *     - premiered     : string (2005-03-04)
        *     - status        : string (Continuing) - status of a TVshow
-       *     - code          : string (tt0110293) - IMDb code
+       *     - set           : string (Batman Collection) - name of the collection
```

Comment:
"set" and "setid" are also used for JSON and for the infolabel naming (listitem.set and listitem.setid). Introducing "settitle" would make things more confusing.